rust/abacus-base/src/oneline_eyre/print.rs

```diff
@@ -0,0 +1,209 @@
+//! This is a port of the backtrace print.rs code which generates one-line backtraces. A lot of
```

Comment:
is this really the only way to get rust binary backtraces working nicely with gcp? this is like very surprisingly low level code for us to have in an agent. What does the rest of the world do?